reactioncommerce/reaction
# refactor: remove `catalog/publish/products` meteor method, use `publishProductsToCatalog` GQL Mutation instead

Resolves parts of #5529   
Impact: **minor**  
Type: **refactor**

## Issue
`publishContainer` and `productGridContainer` still use the `catalog/publish/products` meteor method to publish products, which in turn uses `getGraphQLContextInMeteorMethod`.

## Solution
Re-write anywhere that calls `catalog/publish/products` to use the `publishProuctsToCatalog` mutation instead, remove the `catalog/publish/products` meteor method, which will remove `getGraphQLContextInMeteorMethod` in this instance.

## Breaking changes
Any custom code that uses the `catalog/publish/products` Meteor method will need to be re-written to use the `publishProductsToCatalog` graphql mutation

## Testing
1. See the `catalog/publish/products` meteor method has been removed
1. Attempt to publish changes to a product to the catalog.
1. See you are successful
